src/stories/Library/cookie-placeholder/cookie-placeholder.scss

```diff
@@ -0,0 +1,14 @@
+.cookie-placeholder {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 20px;
```

Comment:
If we have a variable in the spacing-variables.scss file that is close to this, just use that instead, please. Otherwise it's okay. But we generally prefer using the variables for px values if there is anything close by (+-2px for small things / +- 10px for larger values)

Reply:
I have replaced the padding: 20px and margin:bottom with a suitable variable from variables-spacing.scss.